src/store/transaction/transaction.thunks.js

```diff
@@ -485,6 +478,9 @@ function transfer(amount, from, to, fee) {
     const {
       global: { wallet, coordinatorStateTask },
     } = getState();
+
+    dispatch(transactionActions.startTransactionSigning());
```

Comment:
This is also an L2 tx, and signing is done with Metamask for L1 txs

Reply:
I'm reusing that function to be aware in the view if the transaction fail and reactivate the btn. Maybe I can rename that action to startTransaction and endTransaction

Reply:
I would suggest something like that, yes. Let's see what @elias-garcia thinks

Reply:
I agree with @AlbertoElias here, maybe `startTxApproval`? What do you think guys, I think that it's more generic than `startTxSigning`?

Reply:
I'm still thinking that maybe `startTransaction` is more generic than `startTxApproval` because in L2 you don't need more user steps

Reply:
Thing is, when you start a transaction might be considered the transaction form. I think Approval is a good generic word.